### DISCUSSION TITLE
Password policy

### DISCUSSION THREAD

**Discussion by jorgeimar** [gravitational/teleport#33191] [General]
Hello, it is possible to configure a password policy for local users on teleport? if not, it is on the road map?. This is crucial for us to consider use teleport on production, it is a great tool!! but i havent found anything about set a password policy (password lenght, rotation, etc.)

**Comment by webvictim** [comment]
Teleport does currently enforce a minimum password length limit on local users but this isn't documented or officially exposed as a parameter that can be set. For production usage, we recommend mapping users from SSO providers - Github, OIDC or SAML - and managing password policy there.
